src/components/Header.js

```diff
@@ -41,6 +43,14 @@ export default function(store){
         }
       }
     },
+    mounted() {
```

Comment:
(Might have to also add an event listener for `touchmove`, e.g. http://stackoverflow.com/a/9787064)